### PR TITLE
Format everything with Black

### PR DESCRIPTION
**Pull request by rht**
(https://github.com/psf/black)

### CODE REVIEW

**Comment by JOJ0** [comment]
No, please don't! All the code recently was refactored according to PEP8 in PR#6 (using pylint and he might also have used flake8). I think it's fine already. What exactely are your concerns with the current formatting?

File: synadm/api.py [hunk 1]
```diff
@@ -110,33 +114,34 @@ def user_modify(self, user_id, password, display_name, threepid,
...
+        """Get a list of room members"""
         return self.query("get", f"v1/rooms/{room_id}/members")
 
-    def room_delete(self, room_id, new_room_user_id, room_name, message,
-                    block, no_purge):
-        """ Delete a room and purge it if requested
-        """
+    def room_delete(
+        self, room_id, new_room_user_id, room_name, message, block, no_purge
+    ):
```
Comment (JOJ0) [inline synadm/api.py:140]: For example, this is awful IMHO, the version on the left is properly readable already and spare a line. Using a magic auto-formatting tool is not always the best solution. Still: IMHO, Thanks for the effort anyway :-) I currently like to use recommendations provided by a tool called flake8.

**Comment by rht** [comment]
I think it is whether you like Black or you hate Black. What I did was just to apply Black by default. The point is that `Black` is like `gofmt` where there is only 1 true style to end all the formatting wars. If you haven't heard of it, see https://github.com/psf/black#used-by.

**Comment by JOJ0** [comment]
Yeah it does make sense but I think I am not ready for it at this point. Some things I think get rather longish horizontally. Ask me in a couple of months again. I might change my mind! :-) But definitely thanks anyway! Appreciated!

**Pull request closed without merging**